test.py

```
@@ -0,0 +1,118 @@
+async def go():
+    print("starting test")
+    load_dotenv()
+    session = next(get_db())
+    user_id = "0tyA7AEi9KbHYRyLu33qV9BjPgg1"
+
+    llm_provider = ProviderService(session, user_id)
+    tools_provider = ToolService(session, user_id)
+    prompt_provider = PromptService(session)
+    project_id = "0194d052-19b2-7b01-82a0-299da1538355"
+
+    agent = CrewAIRagAgent(
```

Ruff (0.8.2)

Local variable `agent` is assigned to but never used

Remove assignment to unused variable `agent`

(F841)